lib/features.js

```diff
@@ -21,7 +21,7 @@ const features = {
         packages: [
             { name: 'sass-loader', enforce_version: true },
             // allow node-sass or sass to be installed
-            [{ name: 'sass' }, { name: 'node-sass' }]
+            [{ name: 'sass' }, { name: 'sass-embedded' }, { name: 'node-sass' }]
```

Comment:
please update the comment above; or better, remove it - line intention is clear

Reply:
I removed the comment